scripts/qa_crowbarsetup.sh

```diff
@@ -1754,6 +1754,15 @@ function onadmin_post_allocate
 {
     pre_hook $FUNCNAME
 
+    # NOTE(bbobrov): this is a temporary workaround for SOC-11110;
+    # remove when bsc#1166529 is fixed and the fix is available for us
+    echo "SOC-11110 workaround: enabling dm-event.socket and lvm2-monitor.service"
+    for node in $(get_all_discovered_nodes); do
```

Comment:
I'm wondering if it's necessary to go through all `get_all_discovered_nodes` again in this place. Can't you use some of the existing loops? Maybe the one at the end of `onadmin_allocate`?

Reply:
done